boards/hamilton/board.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2016 University of California, Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_hamilton
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the Hamilton mote
+ *
+ * @author      Michael Andersen <m.andersen@berkeley.edu>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "board.h"
+#include "periph/gpio.h"
+
+const uint64_t* const fb_sentinel     = ((const uint64_t* const)0x3fc00);
```

Comment:
What are these variables for ?

Reply:
@immesys, please answer to this question